content/en/integrations/guide/deprecated-oracle-integration.md

```diff
@@ -0,0 +1,511 @@
+
+Optionally use the `tags` parameter to apply a list of tags to each metric collected.
+
+The following:
+
+```python
+self.gauge('oracle.custom_query.metric1', value, tags=['tester:oracle', 'tag1:value'])
+self.count('oracle.custom_query.metric2', value, tags=['tester:oracle', 'tag1:value'])
+```
+
+is what the following example configuration would become:
+
+```yaml
+- query: | # Use the pipe if you require a multi-line script.
+    SELECT columns
+    FROM tester.test_table
+    WHERE conditions
+  columns:
+    # Put this for any column you wish to skip:
+    - {}
+    - name: metric1
+      type: gauge
+    - name: tag1
+      type: tag
+    - name: metric2
+      type: count
+  tags:
+    - tester:oracle
+```
+
+See the [sample oracle.d/conf.yaml][5] for all available configuration options.
+
+### Example
+
+Create a query configuration to help identify database locks:
+
+1. To include a custom query, modify `conf.d\oracle.d\conf.yaml`. Uncomment the `custom_queries` block, add the required queries and columns, and restart the Agent.
+
+```yaml
+  init_config:
+  instances:
+      - server: localhost:1521
+        service_name: orcl11g.us.oracle.com
+        username: datadog
+        password: xxxxxxx
+        jdbc_driver_path: /u01/app/oracle/product/11.2/dbhome_1/jdbc/lib/ojdbc6.jar
+        tags:
+          - db:oracle
+        custom_queries:
+          - query: |
+              select blocking_session, username, osuser, sid, serial# as serial, wait_class, seconds_in_wait
+              from v_$session
+              where blocking_session is not NULL order by blocking_session
+            columns:
+              - name: blocking_session
+                type: gauge
+              - name: username
+                type: tag
+              - name: osuser
+                type: tag
+              - name: sid
+                type: tag
+              - name: serial
+                type: tag
+              - name: wait_class
+                type: tag
+              - name: seconds_in_wait
+                type: tag
+```
+
+2. To access `v_$session`, give permission to `DATADOG` and test the permissions.
+
+```text
+SQL> grant select on sys.v_$session to datadog;
+
+##connecting with the DD user to validate the access:
+
+
+SQL> show user
+USER is "DATADOG"
+
+
+##creating a synonym to make the view visible
+SQL> create synonym datadog.v_$session for sys.v_$session;
+
+
+Synonym created.
+
+
+SQL> select blocking_session,username,osuser, sid, serial#, wait_class, seconds_in_wait from v_$session
+where blocking_session is not NULL order by blocking_session;
+```
+
+3. Once configured, you can create a [monitor][10] based on `oracle.custom_query.locks` metrics.
+
+## Data Collected
```

Comment:
The "Data Collected" section feels out of scope for the page, I recommend deleting it. They can read about the integration on its dedicated page.